Automating the release process

### Description

The release job in travis is failing to push docker images out in the release process.  The user credentials for the Docker reigstry must have changed, and were not updated in the encrypted Travis configuration

https://travis-ci.org/github/pravega/zookeeper-operator/jobs/736353355

```
Error response from daemon: Get https://registry-1.docker.io/v2/: unauthorized: incorrect username or password
Makefile:84: recipe for target 'login' failed
make: *** [login] Error 1
Script failed with status 2
```

### Importance

❗ Critical ❗ 

### Location

https://github.com/pravega/zookeeper-operator/blob/master/.travis.yml#L12

### Suggestions for an improvement

Fix the credentials to allow docker push

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -9,7 +9,10 @@ on:
   pull_request:
     branches:
       - master
-
+  release:
+    types:
+      - created
+      
 # jobs to run
 jobs:
   build:

--- a/Makefile
+++ b/Makefile
@@ -97,12 +97,11 @@ login:
 test-login:
 	echo "$(DOCKER_TEST_PASS)" | docker login -u "$(DOCKER_TEST_USER)" --password-stdin
 
-push: build-image build-zk-image build-zk-image-swarm login
+push: build-image build-zk-image login
 	docker push $(REPO):$(VERSION)
 	docker push $(REPO):latest
 	docker push $(APP_REPO):$(VERSION)
 	docker push $(APP_REPO):latest
-	docker push $(APP_REPO):$(VERSION)-swarm
 	docker tag $(REPO):$(VERSION) $(ALTREPO):$(VERSION)
 	docker tag $(REPO):$(VERSION) $(ALTREPO):latest
 	docker tag $(APP_REPO):$(VERSION) $(APP_ALTREPO):$(VERSION)

